packages/headless/src/features/facets/facet-set/facet-set-slice.ts

```diff
@@ -63,6 +63,21 @@ export const facetSetReducer = createReducer(
       })
       .addCase(restoreSearchParameters, (state, action) => {
         const f = action.payload.f || {};
+
+        const generatedKeys = Object.keys(f).filter((key) =>
+          key.startsWith('generated_')
+        );
+
+        generatedKeys.forEach((id) => {
+          const field = id.replace(/^generated_/, '');
+          const request = buildFacetRequest({
+            facetId: id,
+            field,
+          });
+          request.generated = true;
+          state[id] = getFacetSetSliceInitialState(request);
+        });
+
```

Comment:
this is the fix for the url-manager. Basically creating facets slice for the generated facets contained in the url